### DISCUSSION TITLE
New adapter

### DISCUSSION THREAD

**Discussion by rockofox** [vrtmrz/obsidian-livesync#205] [General]
Should we use the new adapter? Are there any known issues with it? Why isn't it default?

**Comment by vrtmrz** [comment]
Recommended, but not a must.

The new adapter uses the database more efficiently. However, it has no compatibility with the previous ones, and we have to migrate existing data. Therefore, if this would be configured as default, some of us might consume time in an unexpected and unwanted situation. (Yes, I did let you all do this once).

However, your question is also plausible. I will consider issuing a confirmation dialogue! Thank you so much!
